src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -20,6 +21,47 @@ const dispatchMap = new Map([
 
 browser.runtime.onMessage.addListener(command => dispatchMap.get(command)?.())
 
+const enhanceNavigation = (event: KeyboardEvent) => {
+    const isEditingTitle = event.target?.parentElement instanceof HTMLHeadingElement
+    const isNoBlockActive = !Roam.getActiveRoamNode()
+    const isTopBlockActive = getActiveEditElement() === getFirstTopLevelBlock()
+    const isInSearch = event.target instanceof HTMLInputElement
+
+    // Don't mess with arrow nav in search results
+    if (isInSearch) return;
+
+    switch (event.key) {
+        case 'Enter':
+            if (isEditingTitle || isNoBlockActive) {
+                Roam.activateTopBlock()
+            }
+            break
+        case 'ArrowUp':
+            if (isNoBlockActive || isTopBlockActive) {
+                if (event.getModifierState("Shift")) {
```

Comment:
this seems indicative of potentially larger surface area of impact (e.g. this is gonna break alt+up or cmd+up/etc in top block, maybe other things).   
Makes me hesitate about overriding things that are so fundamental. How about just adding a shortcut to edit the title?

Reply:
I take your point, and am considering extending the modifier check to include the full set of modifiers listed [here](https://www.w3.org/TR/uievents-key/#keys-modifier) so that it's impossible for there to be a conflict with existing shortcuts.

Reply:
I'm still not entirely sure. seems like a rather intimate set of things to override and things can go wrong... that said, I think it can still be ok to include it. assuming we have a way to disable it and do careful testing that we don't introduce unexpected behaviors.   
What about just scrolling the page with the keyboard? (I don't use it as I use vim mappings, but seems plausibly like something people use?)